Aspose.Slides.WebExtensions/Helpers/TextHelper.cs

```diff
@@ -270,7 +272,8 @@ public static string GetTextStyle<T>(IPortionFormatEffectiveData format, ITextFr
         public static string GetTextStyle<T>(IParagraphFormatEffectiveData format, TemplateContext<T> model)
         {
             string alignment = TextHelper.GetHorizontalAlignmentStyle(format.Alignment);
-            string lineSpacingStyle = TextHelper.GetLineSpacingStyle(format);
+            float fontHeight = (model.Object as Paragraph).Portions[0].PortionFormat.GetEffective().FontHeight;

```

Comment:
@polonair ,

Why not to use direct case, i.e. "((Paragraph)model.Object)"? I mean, here we choosing between NRE and InvalidCast and the last one looks better imo

Reply:
Yes, I agree with you.

Reply:
[looks like it's better not to use generics in this case](https://github.com/aspose-slides/Aspose.Slides.WebExtensions/pull/7/commits/c2345af07c654ba57a1d652a81f449394bdc2a44)